src/zcbenchmarks.cpp

```diff
@@ -42,64 +44,52 @@ double benchmark_sleep()
 double benchmark_parameter_loading()
 {
     // FIXME: this is duplicated with the actual loading code
-    boost::filesystem::path pk_path = ZC_GetParamsDir() / "zc-testnet-public-alpha-proving.key";
-    boost::filesystem::path vk_path = ZC_GetParamsDir() / "zc-testnet-public-alpha-verification.key";
+    boost::filesystem::path pk_path = ZC_GetParamsDir() / "z3-proving.key";
+    boost::filesystem::path vk_path = ZC_GetParamsDir() / "z3-verification.key";
 
     timer_start();
-    auto vk_loaded = libzerocash::ZerocashParams::LoadVerificationKeyFromFile(
-        vk_path.string(),
-        INCREMENTAL_MERKLE_TREE_DEPTH
-    );
-    auto pk_loaded = libzerocash::ZerocashParams::LoadProvingKeyFromFile(
-        pk_path.string(),
-        INCREMENTAL_MERKLE_TREE_DEPTH
-    );
-    libzerocash::ZerocashParams zerocashParams = libzerocash::ZerocashParams(
-        INCREMENTAL_MERKLE_TREE_DEPTH,
-        &pk_loaded,
-        &vk_loaded
-    );
-    return timer_stop();
-}
 
-double benchmark_create_joinsplit()
```

Comment:
I was confused because the diff is kinda wonky here, the `+` lines are still part of `benchmark_parameter_loading()`.